ExtraDry/ExtraDry.Blazor/Components/Standard/Suspense.razor.scss

```diff
@@ -15,3 +15,7 @@
 ::deep i.large {
     font-size: 28px;
 }
+
```

Comment:
Another spot where we're making it difficult for users to override style.  Can you please add @layer extra-dry to this file as well?